File: batch1/src/Address.java

public class Address {

	private String city;
	private String state;
	private int pin;
	
	public Address(String city, String state, int pin) {
		super();
		this.city = city;
		this.state = state;
		this.pin = pin;
	}

	public String getCity() {
		return city;
	}

	public String getState() {
		return state;
	}

	public int getPin() {
		return pin;
	}
	
	
	
}

File: web/react/react_older/reactcfg_sep_24/2025 aug/my-app/src/employee/address.js
import { Component } from "react";

class Address extends Component {
    constructor() {
        super()
        this.state = {
            "addr_id": "add100001",
            "city": "new delhi",
            "state": "delhi",
        }
    }

    render() {
        return (
            <div className="bg-primary text-info pt-5 w-50 border border-secondary"
                style={{ "marginLeft": "100pt", "marginTop": "10pt", "padding": "20pt", "border-radius": "40pt" }} >
                this is address class
                <div>id {this.state.addr_id}</div>
                <div>city {this.state.city}</div>
                <div>state{this.state.state}</div>

                <div>
                    <button className="btn btn-light" onClick={this.changeAddress}>Change Address</button>
                </div>
            </div>
        )
    }

    changeAddress=()=>{
        this.setState({
            "addr_id": "add100002",
            "city": "cuttak",
            "state": "odisha",
        })
    }
}
export default Address;